src/platforms/hubspot/actions/calls/create.ts

```diff
@@ -0,0 +1,41 @@
+import { z } from 'zod';
+import { action } from '@/sdk';
+import client from '../../client';
+import { transformCall } from '@/platforms/hubspot/actions/mappers';
+
+export default action(
+  'calls-create',
```

Comment:
did you end up trying to change the types to remove action names and just use operation + resource?